### PR TITLE
Story/scat 5396 DA enter your weightings page and sub contractor page

### PR DESCRIPTION
**Pull request by dhivya-cognizant**
### JIRA link
https://crowncommercialservice.atlassian.net/browse/SCAT-5396

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Review and publish page updated
- [ ] UI changes look good on mobile

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/requirements/controller/da-SubContractors.ts [hunk 1]
```diff
@@ -98,19 +93,20 @@ export const DA_POST_SUBCONTRACTORS = async (req: express.Request, res: express.
           weighting: dimension.weighting,
           requirements: dimension.requirements,
           includedCriteria: dimension.includedCriteria.map(criteria => {
-            if (!req.session['CapAss']?.isSubContractorAccepted && criteria['name'] == 'Sub Contractor') {
+            if (!da_acceptsubcontractors && criteria['name'] == 'Sub Contractor') {
```
Comment (BalwinderSinghCts) [inline src/main/features/requirements/controller/da-SubContractors.ts]: 1. Use ToLower or Toupper function

File: src/main/features/requirements/controller/da-SubContractors.ts [hunk 2]
```diff
@@ -89,7 +83,8 @@ export const DA_POST_SUBCONTRACTORS = async (req: express.Request, res: express.
     const { da_subContractors } = req.body;
 
     if (da_subContractors !== undefined && da_subContractors !== '') {
-      req.session['CapAss']?.isSubContractorAccepted = da_subContractors == 'yes' ? true : false;
+      const da_acceptsubcontractors = da_subContractors == 'yes' ? true : false;
```
Comment (BalwinderSinghCts) [inline src/main/features/requirements/controller/da-SubContractors.ts]: 1. Use ToLower or Toupper function

Comment (dhivya-cognizant) [reply src/main/features/requirements/controller/da-SubContractors.ts]: I have implemented it

File: src/main/features/requirements/controller/da-SubContractors.ts (hunk 1 above)
Comment (dhivya-cognizant) [reply src/main/features/requirements/controller/da-SubContractors.ts]: I have implemented it

**Review by SahitiVemulapalli — APPROVED** [review]

**Review by 2073810-Sonam — APPROVED** [review]

**Pull request merged**